### PR TITLE
Mich/mkr proposal

### PR DESCRIPTION
**Pull request by Maltmark**
change MKR LT LTV in V2
https://snapshot.org/#/aave.eth/proposal/0x9f2c87a0eb6296ca1ef45bc3ce70cbd7884a0c7b960f17f32a0742abf89c2b8a

### CODE REVIEW

**Comment by ori-chaoslabs** [comment]
numbers look good to me

File: src/payloads/MKRPayload-Feb26.sol [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+/**
+ * @title Chaos <> AAVE Proposal
+ * @author Chaos
+ * @notice MKR LT, LTV changes
+ * Governance Forum Post: https://governance.aave.com/t/arc-chaos-labs-risk-parameter-updates-mkr-on-aave-v2-ethereum-2023-02-17/11948
+ * Snapshot: https://snapshot.org/#/aave.eth/proposal/0x9f2c87a0eb6296ca1ef45bc3ce70cbd7884a0c7b960f17f32a0742abf89c2b8a
+ */
+contract ProposalPayload {
+    address public constant MKR = 0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2;
```
Comment (yhayun) [inline src/payloads/MKRPayload-Feb26.sol]: This should be a const from address book

File: script/specific/DeployMKR-Feb26ProposalPayload.s.sol [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import "@forge-std/console.sol";
+import {Script} from "@forge-std/Script.sol";
+import {ProposalPayload} from "../../src/payloads/MKRPayload-Feb26.sol";
+
+contract DeployProposalPayload is Script {
+    function run() external {
+        vm.startBroadcast();
+        ProposalPayload proposalPayload = new ProposalPayload();
+        console.log("Proposal Payload address", address(proposalPayload));
```
Comment (yhayun) [inline script/specific/DeployMKR-Feb26ProposalPayload.s.sol]: No need to console, broadcast action will log all needed info

**Review by yhayun — APPROVED** [review]

File: script/specific/DeployMKR-Feb26ProposalPayload.s.sol (hunk 2 above)
Comment (Maltmark) [reply script/specific/DeployMKR-Feb26ProposalPayload.s.sol]: done

Comment (yhayun) [reply script/specific/DeployMKR-Feb26ProposalPayload.s.sol]: I still see the console here. ?

**Review by yhayun — APPROVED** [review]

File: script/specific/DeployMKR-Feb26ProposalPayload.s.sol (hunk 2 above)
Comment (Maltmark) [reply script/specific/DeployMKR-Feb26ProposalPayload.s.sol]: no more :)

**Pull request merged**